### ISSUE TITLE
[Issue]: Installtion haulted: The specified module could not be found. c10_xpu.dll

### ISSUE THREAD

**Issue by moongear** [vladmandic/automatic#3553]
### Issue Description

**Any ideas on the below error?**

Inatalling latest verion from repo via clone.  I can run .\webui.bat without --use-ipex and it works, just no GPU.

Maybe worth noting, I have the latest FPU driver from Intel (not Micro$oft) and have the OneAPI toolkit installed.

Machine OS
OS Name	Microsoft Windows 11 Pro
Version	10.0.26100 Build 26100

Machine Memory
Installed Physical Memory (RAM)	96.0 GB
Total Physical Memory	95.6 GB
Available Physical Memory	83.3 GB
Total Virtual Memory	109 GB
Available Virtual Memory	96.8 GB

GPU
Name	Intel(R) Arc(TM) Graphics
PNP Device ID	PCI\VEN_8086&DEV_7D55&SUBSYS_80092014&REV_08\3&11583659&0&10
Adapter Type	Intel(R) Arc(TM) Graphics Family, Intel Corporation compatible
Adapter Description	Intel(R) Arc(TM) Graphics
Adapter RAM	2.00 GB (2,147,479,552 bytes)

(Full hardware details below)

I thought this would work but maybe it won't.  I did get Ollama working via the GPU which significantly improved speed.  Any ehlp or advice is appreacited.

### Version Platform Description

**Machine General:**

OS Name	Microsoft Windows 11 Pro
Version	10.0.26100 Build 26100
Other OS Description 	Not Available
OS Manufacturer	Microsoft Corporation
System Name	AI_MACHINE
System Manufacturer	GMKtec
System Model	NucBox K9
System Type	x64-based PC
System SKU	K9-001
Processor	Intel(R) Core(TM) Ultra 5 125H, 3600 Mhz, 14 Core(s), 18 Logical Processor(s)
BIOS Version/Date	American Megatrends International, LLC. NucBox K9 1.22, 5/9/2024
SMBIOS Version	3.6
Embedded Controller Version	1.08
BIOS Mode	UEFI
BaseBoard Manufacturer	GMKtec
BaseBoard Product	GMKtec
BaseBoard Version	V1.0
Platform Role	Mobile
Secure Boot State	Off
PCR7 Configuration	Elevation Required to View
Windows Directory	C:\Windows
System Directory	C:\Windows\system32
Boot Device	\Device\HarddiskVolume2
Locale	United States
Hardware Abstraction Layer	Version = "10.0.26100.1"
User Name	Not Available
Time Zone	Eastern Daylight Time
Installed Physical Memory (RAM)	96.0 GB
Total Physical Memory	95.6 GB
Available Physical Memory	83.3 GB
Total Virtual Memory	109 GB
Available Virtual Memory	96.8 GB
Page File Space	13.0 GB
Page File	C:\pagefile.sys
Kernel DMA Protection	On
Virtualization-based security	Running
Virtualization-based security Required Security Properties	Base Virtualization Support
Virtualization-based security Available Security Properties	Base Virtualization Support, DMA Protection, UEFI Code Readonly, SMM Security Mitigations 1.0, Mode Based Execution Control, APIC Virtualization
Virtualization-based security Services Configured	Hypervisor enforced Code Integrity
Virtualization-based security Services Running	Hypervisor enforced Code Integrity
App Control for Business policy	Enforced
App Control for Business user mode policy	Audit
Automatic Device Encryption Support	Elevation Required to View

**GPU**

Name	Microsoft Remote Display Adapter
PNP Device ID	SWD\REMOTEDISPLAYENUM\RDPIDD_INDIRECTDISPLAY&SESSIONID_0002
Adapter Type	Not Available, Microsoft compatible
Adapter Description	Microsoft Remote Display Adapter
Adapter RAM	Not Available
Installed Drivers	Not Available
Driver Version	10.0.26100.1150
INF File	rdpidd.inf (RdpIdd_Install.NT section)
Color Planes	Not Available
Color Table Entries	4294967296
Resolution	1920 x 1080 x 32 hertz
Bits/Pixel	32
Driver	C:\WINDOWS\SYSTEM32\DRIVERS\WUDFRD.SYS (10.0.26100.1882, 353.42 KB (361,904 bytes), 10/4/2024 8:18 PM)
	
Name	Intel(R) Arc(TM) Graphics
PNP Device ID	PCI\VEN_8086&DEV_7D55&SUBSYS_80092014&REV_08\3&11583659&0&10
Adapter Type	Intel(R) Arc(TM) Graphics Family, Intel Corporation compatible
Adapter Description	Intel(R) Arc(TM) Graphics
Adapter RAM	2.00 GB (2,147,479,552 bytes)
Installed Drivers	C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_181a492eed27d42f\igd9trinity64.dll,C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_181a492eed27d42f\igd10iumd64.dll,C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_181a492eed27d42f\igd10iumd64.dll,C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_181a492eed27d42f\igd12umd64.dll
Driver Version	32.0.101.6078
INF File	oem45.inf (MTL_IAG_wNext section)
Color Planes	Not Available
Color Table Entries	4294967296
Resolution	1920 x 1080 x 60 hertz
Bits/Pixel	32
Memory Address	0x17000000-0x17FFFFFF
Memory Address	0x0000-0xFFFFFFF
IRQ Channel	IRQ 4294967204
Driver	C:\WINDOWS\SYSTEM32\DRIVERSTORE\FILEREPOSITORY\IIGD_DCH.INF_AMD64_181A492EED27D42F\IGDKMDN64.SYS (32.0.101.6078, 50.42 MB (52,871,528 bytes), 11/2/2024 2:50 PM)

### Relevant log output

```shell
17:56:46-041026 INFO     Command line args: ['--use-ipex'] use_ipex=True
┌───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────┐
│ C:\ai\sdnext\automatic\launch.py:274 in <module>                                                                    │
│                                                                                                                     │
│   273 if __name__ == "__main__":                                                                                    │
│ > 274     main()                                                                                                    │
│   275                                                                                                               │
│                                                                                                                     │
│ C:\ai\sdnext\automatic\launch.py:251 in main                                                                        │
│                                                                                                                     │
│   250                                                                                                               │
│ > 251     uv, instance = start_server(immediate=True, server=None)                                                  │
│   252     while True:                                                                                               │
│                                                                                                                     │
│ C:\ai\sdnext\automatic\launch.py:167 in start_server                                                                │
│                                                                                                                     │
│   166     installer.log.debug(f'Starting module: {server}')                                                         │
│ > 167     module_spec.loader.exec_module(server)                                                                    │
│   168     uvicorn = None                                                                                            │
│ in exec_module:883                                                                                                  │
│ in _call_with_frames_removed:241                                                                                    │
│                                                                                                                     │
│ C:\ai\sdnext\automatic\webui.py:11 in <module>                                                                      │
│                                                                                                                     │
│    10 from threading import Thread                                                                                  │
│ >  11 import modules.loader                                                                                         │
│    12 import torch # pylint: disable=wrong-import-order                                                             │
│                                                                                                                     │
│ C:\ai\sdnext\automatic\modules\loader.py:18 in <module>                                                             │
│                                                                                                                     │
│    17 os.environ.setdefault('TORCH_LOGS', '-all')                                                                   │
│ >  18 import torch # pylint: disable=C0411                                                                          │
│    19 if torch.__version__.startswith('2.5.0'):                                                                     │
│                                                                                                                     │
│ C:\ai\sdnext\automatic\venv\lib\site-packages\torch\__init__.py:271 in <module>                                     │
│                                                                                                                     │
│    270                                                                                                              │
│ >  271     _load_dll_libraries()                                                                                    │
│    272     del _load_dll_libraries                                                                                  │
│                                                                                                                     │
│ C:\ai\sdnext\automatic\venv\lib\site-packages\torch\__init__.py:267 in _load_dll_libraries                          │
│                                                                                                                     │
│    266                     )                                                                                        │
│ >  267                     raise err                                                                                │
│    268                                                                                                              │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
OSError: [WinError 126] The specified module could not be found. Error loading "C:\ai\sdnext\automatic\venv\lib\site-packages\torch\lib\c10_xpu.dll" or one of its dependencies.
Press any key to continue . . .
```

### Backend

Diffusers

### UI

Standard

### Branch

Master

### Model

Other

### Acknowledgements

- [X] I have read the above and searched for existing issues
- [X] I confirm that this is classified correctly and its not an extension issue

**Comment by vladmandic** [comment]
closing as direct duplicate, search issues before creating a new one.

**Comment by moongear** [comment]
Thanks for the fast response!

I just want you to know that I did search. However, happened was, the search feature has "**is:issue is:open**"  (note IS OPEN) defaulted in the search box.  Therefore, my "SEARCH" returned no results.

So, your presumption that I did not search was false.  I hope this helps you help someone else in the future by including this valuable information about the search feature default.

Ultimately, I found the duplicate and am working on the solutions now.  Thank you for the work you do.
